.editorconfig

```diff
@@ -1,12 +1,32 @@
 # EditorConfig is awesome: http://EditorConfig.org
 
 [*]
-indent_style = space
-indent_size = 2
-end_of_line = lf
-charset = utf-8
+indent_style = tab
+indent_size = 1
```

Comment:
I think this will set (visual) indentation in the editor to a single space character - shouldn't this be something like _2_ or _4_ instead?

Reply:
In my understanding this sets indentation as one tab, the visual representation is in the GUI definitions of each editor locally. However since .editorconfig files are just a recommendation and not an official standard, every editor and IDE may decide to interpret this differently. I didn't try this on too many of them (I think I only installed integrations on VIM and VSC so far)

Reply:
If you want me to change it, I'll fix it in #1433 rather than here, of course :)

Reply:
The definition of `indent_size` in the editorconfig documentation (https://editorconfig.org/#file-format-details) looks like this:

> **indent_size**: a whole number defining the number of columns used for each indentation level and the width of soft tabs (when supported). When set to "tab", the value of **tab_width** (if specified) will be used.

To me, that reads like the number of spaces used for indentation, which means one space in the above example.

Reply:
A search of github suggests nobody uses `"1"`

* https://github.com/search?q=%22indent_size+%3D+%22&type=Code

General values are : 2, 3, 4 and 8 (as might be expected, the most)

This also seems to further the idea that the value represents the number of spaces used for indentation.

Reply:
@seefood hahaha, I went down this rabbit hole once. What @nwinkler said is accurate. You'll want to set `tab_width`. I suggest `4`.